source/frontend/src/features/mapSideBar/acquisition/tabs/stakeholders/update/InterestHolderSubForm.tsx

```diff
@@ -0,0 +1,153 @@
+import { FieldArrayRenderProps, FormikErrors, getIn, useFormikContext } from 'formik';
+import * as React from 'react';
+import { Col, Row } from 'react-bootstrap';
+import { FaTrash } from 'react-icons/fa';
+
+import { StyledRemoveLinkButton } from '@/components/common/buttons';
+import { DisplayError, Select } from '@/components/common/form';
+import { SelectOption } from '@/components/common/form';
+import { ContactInputContainer } from '@/components/common/form/ContactInput/ContactInputContainer';
+import ContactInputView from '@/components/common/form/ContactInput/ContactInputView';
+import { SectionField } from '@/components/common/Section/SectionField';
+import FilePropertiesTable from '@/components/filePropertiesTable/FilePropertiesTable';
+import * as API from '@/constants/API';
+import { useOrganizationDetail } from '@/features/contacts/hooks/useOrganizationDetail';
+import useLookupCodeHelpers from '@/hooks/useLookupCodeHelpers';
+import { IEditablePerson } from '@/interfaces/editable-contact';
+import { Api_AcquisitionFile } from '@/models/api/AcquisitionFile';
+import { Api_PropertyFile } from '@/models/api/PropertyFile';
+import { formatFullName } from '@/utils/personUtils';
+
+import { InterestHolderForm, StakeHolderForm } from './models';
+
+export interface IInterestHolderProps {
+  index: number;
+  errors: FormikErrors<StakeHolderForm>;
+  arrayHelpers: FieldArrayRenderProps;
+  file: Api_AcquisitionFile;
+  interestHolder: InterestHolderForm;
+}
+
+export const InterestHolderSubForm: React.FunctionComponent<IInterestHolderProps> = ({
+  index,
+  errors,
+  arrayHelpers,
+  file,
+  interestHolder,
+}) => {
+  const { values, setFieldValue } = useFormikContext<any>();
```

Comment:
prefer avoiding use of any. if this subform is intended for use with a specific interface use that interface instead of any here.